crictl: add page (#8152)
tldr-pages/tldr: pages/common/crictl.md

# crictl

> Command-line for CRI-compatible container runtimes.
> More information: <https://github.com/kubernetes-sigs/cri-tools/blob/master/docs/crictl.md>.

> - List all kubernetes pods(Ready and NotReady):

Reviewer: ```suggestion
- List all kubernetes pods (Ready and NotReady):
```

`crictl pods`

> - List all containers (Running and Exited):

Reviewer: Should be in upper case?
```suggestion
- List all containers (running and exited):
```

Author: > Should be in upper case?

When I executed this command, the result list show me the state is `Running` or  `Exited `, so I thought it would be in upper case. Do you think it's okay?

`crictl ps --all`

- List all images:

`crictl images`

> - Display the status of one or more containers:

Reviewer: is `status` or `information` of a container?
```suggestion
- Display the status of one or more containers:
```

> - Display the information of one or more containers:

Reviewer: ```suggestion
- Display information about one or more containers:
```

> `crictl inspect {{container_id}}`

Reviewer 2: If you mentioned `one or more container` in the description, is it better to change to this:

```suggestion
`crictl inspect {{container_id}} {{container_id}}`
```

> - Display information about one or more containers:
> 
> `crictl inspect {{container_id}} {{container_id}}`

Reviewer 3: ```suggestion
- Print information about specific containers:

`crictl inspect {{container_id1 container_id2 ...}}`
```

Reviewer 3: I think we have to decide what verbs we use: print, display or anything else. What about to stick to `print`/`list`? The second one use for list-like output and the first one for anything else.

> - Open a shell inside a running container:
> 
> `crictl exec -it {{container_id}} {{sh}}`

Reviewer 3: ```suggestion
- Open a specific shell inside a running container:

`crictl exec -it {{container_id}} {{sh}}`
```

> - Pull an image from a registry:
> 
> `crictl pull {{image:tag}}`

Reviewer 3: ```suggestion
- Pull a specific image from a registry:

`crictl pull {{image:tag}}`
```

> - Fetch and follow the logs of a container:

Reviewer: ```suggestion
- Print and follow logs of a container:
```

> - Print and follow logs of a container:
> 
> `crictl logs -f {{container_id}}`

Reviewer 3: ```suggestion
- Print and [f]ollow logs of a specific container:

`crictl logs -f {{container_id}}`
```

> - Remove one or more images:

Reviewer: This command can't remove multiple in `docker` :thinking:

Author: > This command can't remove multiple in `docker` 🤔

@navarroaxel 
When I executed this command in `docker`, it helped me to delete multiple images.
<img width="525" alt="image" src="https://user-images.githubusercontent.com/53329052/175806868-e261cc51-5964-4fb7-a1a2-ace1ce242d77.png">

Reviewer: Oh, I remember this doesn't work. anyway!

> `crictl rmi {{image_id}}`

Reviewer 2: Same here

```suggestion
`crictl rmi {{image_id}} {{image_id}}`
```
